### DISCUSSION TITLE
Difficulty loading some functions

### DISCUSSION THREAD

**Discussion by lilburdeh** [SCasanova/f1dataR#197] [General]
Thanks for creating this package! I'm having trouble with some of the functions. For example, when I write:
```r
load_driver_telemetry(season = "current", round = "last", session = "R", driver = "VER")
``` 
I get: 
```
Error in if (getOption("f1dataR.cache") %in% c("memory", "off", "filesystem")) { : 
  argument is of length zero
```
Any idea how to fix this?

**Comment by pbulsink** [comment]
I think it's being worked on - but to be sure I've created an issue that covers this. We can track progress there :)

**Reply by lilburdeh** [reply]
ok thanks!
